libs/langchain-cohere/src/embeddings.ts

```diff
@@ -137,6 +154,25 @@ export class CohereEmbeddings
     }
   }
 
+  async embed(
```

Comment:
Why add this? Would prefer not to extend base class' API unnecessarily.

Reply:
This method is supported in langchain-python, and also allows to override the inputType parameter to call the embed function